configurer/path_windows.go

```diff
@@ -48,3 +57,18 @@ func DefaultPastelExecutableDir() string {
 	}
 	return filepath.Join(homeDir, filepath.FromSlash(appDir), "PastelWallet")
 }
+
+// GetDownloadPath returns download path of the pastel executables.
+func GetDownloadPath(version string, tool constants.ToolType, architectrue constants.ArchitectureType) string {
```

Comment:
This function is almost exactly the same as in path_linux & path_darwin.
only difference is win, ubuntu, darwin.
You can define & use 1 function in place of these 3